doc/about/users.rst

```diff
@@ -0,0 +1,14 @@
+Users
```

Comment:
Let's remove this file and add this later. It should be internationally oriented as well.

Reply:
There should be international drinking water and engineering companies right? :)

But we should discuss first whether or not we want to add a list of users before adding it so I'll remove it for now.